static/js/components/CourseList.js

```diff
@@ -45,6 +53,18 @@ class CourseList extends React.Component {
         </ul>
         {table}
       </div>;
+
+      let applyForMastersUI = <div className="clear-float">
```

Comment:
Can you leave the JSX here as one declaration, inlined in the value we `return`? there's no control flow happening here so there's not really a need to separate it out like this (ditto for `programListingUI` above)